src/python_routing_v02/build_tests.py

```diff
@@ -226,6 +233,7 @@ def parity_check(parity_parameters, run_parameters, nts, dt, results):
         # construct comparable dataframes
         trt = pd.DataFrame(
             flows.loc[:, compare_node].values,
+            #depths.loc[:, compare_node].values,
```

Comment:
@jameshalgren I put in the option to parity check the depth or water elevation commented out. I was thinking that you would like the structure to remain consistent as it is to produce results easily in flow for either the channel or the reservoir. Should I change things up to have depth be a more consistent check instead of having it set up like this?